middleware/logger.go

```diff
@@ -34,7 +64,7 @@ func Logger() echo.MiddlewareFunc {
 				code = color.Cyan(n)
 			}
 
-			log.Printf("%s %s %s %s %d", method, path, code, stop.Sub(start), size)
+			log.Printf("%s %s %s %s %s %d", parseIP(remoteAddr), method, path, code, stop.Sub(start), size)
```

Comment:
IMHO, this logger, and the other core middleware (except maybe compression) aren't necessarily meant to be used for every production situation. The logs it produces don't contain several that might be needed by some users. It should be kept as simple as possible.

@husobee, out of curiosity, why not use [net.SplitHostPort](http://golang.org/pkg/net/#SplitHostPort)